src/libPOW/pow.cpp

```diff
@@ -47,7 +50,29 @@ POW::POW()
         };
 
         CLMiner::setNumInstances(UINT_MAX);
-        m_miner = std::make_unique<dev::eth::CLMiner>();
+        m_miner = std::make_unique<CLMiner>();
+        LOG_GENERAL(INFO, "OpenCL GPU initialized in POW");
+    }
+    else if (CUDA_GPU_MINE)
+    {
+#ifdef ETHASH_CUDA
+        using namespace dev::eth;
+        CUDAMiner::setNumInstances(UINT_MAX);
+        if (!CUDAMiner::configureGPU(
+                CUDAMiner::c_defaultBlockSize, CUDAMiner::c_defaultGridSize,
+                CUDAMiner::c_defaultNumStreams, 4, 0, 0, false, false))
```

Comment:
Are they values that are configurable (like the number of streams)? If so, can we provide a config file for users to change the setting easily?

Reply:
Yes, i think in next PR i need to create a GPU configuration file to let user set the GPU parameters.